### PR TITLE
upgraded to 7.0.0.Beta8

### PR DESCRIPTION
**Pull request by mbiarnes**

### CODE REVIEW

**Comment by mbiarnes** [comment]
@psiroky can you please ack this - I can megre it, but at least I want one +1

File: download/download.adoc [hunk 1]
```diff
@@ -66,7 +66,7 @@ A CR should be almost identical to the Final release.
 * image:download.png[] Download jBPM *#{site.pom.latest.version}* binaries
 ** #{site.pom.latest.jbpmBinZip}[Download jBPM #{site.pom.latest.version}-bin.zip]
 ** #{site.pom.latest.jbpmExamplesZip}[Download jBPM #{site.pom.latest.version}-examples.zip]
-** #{site.pom.latest.jbpmInstallerZip}[Download jBPM #{site.pom.latest.version}-installer.zip]
+** #{site.pom.latest.jbpmInstallerZip}[Download jBPM installer-#{site.pom.latest.version}.zip]
```
Comment (psiroky) [inline download/download.adoc]: This looks weird:) What about "jBPM installer 7.0.0.Beta.zip" or even keep the previous name to be consistent with the other entries (I don't think it matters that much that the file name actually changed from `jbpm-<version>installer.zip` to `jbpm-installer-<version>.zip`)

Comment (mbiarnes) [inline download/download.adoc]: @psiroky I saw when uploading the binaries to filemgmt.jboss.org that the jbpm-installer changes its name: from jbpm-<version>-installer.zip to jbpm-installer-<version>.zip - so I thought maybe they want to have this file another name

**Comment by mbiarnes** [comment]
@psiroky could you please ack this? so we can merge it before 7.0.0.CR1 is out

**Pull request merged**